docs/Plugins.md

```diff
@@ -1,36 +1,39 @@
 Plugins
 =======
 
-Moya plugins are used to perform some side effect whenever a request is sent or received. Plugins are defined by the `PluginType` protocol, which can receive callbacks when:
-- a request is going to be sent;
+Moya plugins are used modify requests and responses or perform side-effects. A
+plugin is called when:
+- a request is being prepared.
```

Comment:
Hmm, dunno about this "when request is being prepared". Should the user be worried that Moya is still preparing the request? I'd go with something that describes it more clearly. Your docs for functions are good enough imo.